cardano-testnet/src/Testnet/Defaults.hs

```diff
@@ -72,32 +69,25 @@ import           Numeric.Natural
 import           System.FilePath ((</>))
 
 import           Test.Cardano.Ledger.Core.Rational
+import           Test.Cardano.Ledger.Plutus (testingCostModelV3)
 import           Testnet.Start.Types
 import           Testnet.Types
 
 {- HLINT ignore "Use underscore" -}
 
 instance Api.Error AlonzoGenesisError where
-  prettyError (AlonzoGenErrCostModels e) =
-    "Error in Alonzo genesis cost models: " <> pshow e
   prettyError (AlonzoGenErrTooMuchPrecision r) =
     "Too much precision for bounded rational in Alonzo genesis: " <> pshow r
 
-data AlonzoGenesisError
+newtype AlonzoGenesisError
   = AlonzoGenErrTooMuchPrecision Rational
-  | AlonzoGenErrCostModels (Map Ledger.Language Ledger.CostModelError)
   deriving Show
 
 defaultAlonzoGenesis :: Either AlonzoGenesisError AlonzoGenesis
 defaultAlonzoGenesis = do
   let genesis = Api.alonzoGenesisDefaults
-      costModelsErrors = Ledger.costModelsErrors $ Ledger.agCostModels genesis
```

Comment:
What's the right way to do this now?

Reply:
My understanding is that the cost model will simply fail to parse if it's invalid. Here's the `cardano-ledger` side change: https://github.com/IntersectMBO/cardano-ledger/commit/e173cf221e946f1e0d5ad1b26d54f3b65e740c39
We just remove it there as well. I also asked @lehins when I was doing it if that's right, here's his response:
> `CostModelError can completely be removed. This is because we now in ledger turn into a decoder error, so noone really need to deal with it anymore